### DISCUSSION TITLE
Airflow Self-heal

### DISCUSSION THREAD

**Discussion by Praful19** [apache/airflow#21629] [Q&A]
Hi 

I have airflow where there are Mutiple Dags are running. I am looking for ideas/suggestion how we can automate Airflow so that it can handle errors whenever there is failure of certain Dags. Kindly suggest 

Regard
Praful

**Comment by potiuk** [comment]
All the options are described in the docs: https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/index.html
